### DISCUSSION TITLE
[Vote ended on 2023-08-02] Remove ngine_io.vultr from Ansible 9

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#259] [Votes done]
The [ngine_io.vultr](https://github.com/ngine-io/ansible-collection-vultr) has been archived. Additionally, it looks like it's based on the Vultr v1 API which will be shut down after August 14th, so it won't be useful / usable after this date.

Please vote with +1 to remove it from Ansible 9 or -1 to keep it.

If you want to discuss this, please use #257.

**Comment by mariolenz** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by resmo** [comment]
as collection maintainer:
+1

**Comment by felixfontein** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by thaumos** [comment]
No vote.
